tests/compute/credentials.go

```diff
@@ -53,82 +53,91 @@ var _ = SIGDescribe("Guest Access Credentials", func() {
 		guestAgentConnetTimeout = 2 * time.Minute
 		denyListTimeout         = 2 * time.Minute
 		fedoraPassword          = "fedora"
+		pubKeySecretID          = "my-pub-key"
+		userPassSecretID        = "my-user-pass"
 	)
 
-	Context("with qemu guest agent", func() {
-		withQuestAgentPropagationMethod := v1.SSHPublicKeyAccessCredentialPropagationMethod{
-			QemuGuestAgent: &v1.QemuGuestAgentSSHPublicKeyAccessCredentialPropagation{
-				Users: []string{"fedora"},
-			},
-		}
-
-		withPassword := func(secretName string) libvmi.Option {
-			return func(vmi *v1.VirtualMachineInstance) {
-				vmi.Spec.AccessCredentials = append(vmi.Spec.AccessCredentials,
-					v1.AccessCredential{
-						UserPassword: &v1.UserPasswordAccessCredential{
-							Source: v1.UserPasswordAccessCredentialSource{
-								Secret: &v1.AccessCredentialSecretSource{
-									SecretName: secretName,
-								},
-							},
-							PropagationMethod: v1.UserPasswordAccessCredentialPropagationMethod{
-								QemuGuestAgent: &v1.QemuGuestAgentUserPasswordAccessCredentialPropagation{},
-							},
-						},
-					})
-			}
-		}
+	keysSecretData := libsecret.DataBytes{
+		"my-key1": []byte("ssh-rsa AAAAB3NzaC1yc2EAAAABIwAAAQEA6NF8iallvQVp22WDkT test-ssh-key1"),
+		"my-key2": []byte("ssh-rsa AAAAB3NzaC1yc2EAAAABIwAAAQEA6NF8iallvQVp22WDkT test-ssh-key2"),
+		"my-key3": []byte("ssh-rsa AAAAB3NzaC1yc2EAAAABIwAAAQEA6NF8iallvQVp22WDkT test-ssh-key3"),
+	}
 
-		It("[test_id:6220]should propagate public ssh keys", func() {
-			const secretID = "my-pub-key"
-			vmi := libvmifact.NewFedora(withSSHPK(secretID, withQuestAgentPropagationMethod))
+	userData := fmt.Sprintf(
```

Comment:
Can you make this a const and remove the part setting the password? In test case `using qemu guest agent` you are not setting the password and still it is possible to login in the Expecter before (Password `fedora` is already the default?)